src/pyproject.toml

```diff
@@ -54,6 +54,7 @@ max-line-length = 88
 ignore  = ['F401', 'W503']
 per-file-ignores = [
     'bolt_app.py:E402',
+    'tests/*:F811'
```

Comment:
Instead of suppressing this warning for all lines in test files, individually suppress it on the lines of code that flake8 complains about fixture names. That way you will still get legitimate "unused import" warnings in the rest of the test code.